grails-app/views/shared/_listAllDocuments.gsp

```diff
@@ -0,0 +1,185 @@
+<div id="projectResources" class="my-4 my-md-5">
+
+    <div id="${containerId}" class="container-fluid">
+
+        <h2>${hubConfig.getTextForResources(grailsApplication.config.content.defaultOverriddenLabels)}</h2>
+
+        <div class="input-group col-12 search-resources">
+            <label for="searchResources" class="sr-only">Search Resources</label>
+            <input class="form-control" id="searchResources" type="text" data-bind="value:searchDoc, hasFocus: searchHasFocus, valueUpdate:'keyup'"
+                   placeholder="Search ${hubConfig.getTextForResources(grailsApplication.config.content.defaultOverriddenLabels)}..." aria-label="Search documents..." aria-describedby="org-search-button"/>
+
+            <div class="input-group-append">
+                <label for="searchType" class="sr-only">Filter by</label>
+                <select id="searchType" class="custom-select" data-bind="options: documentFilterFieldOptions, value: documentFilterField, optionsText: 'label'" aria-label="Filter">
+                </select>
+            </div>
+
+            <div class="col-md-4 text-right mt-2 mt-md-0">
+                <label for="sortBy" class="col-form-label">Sort by</label>
+                <select id="sortBy" class="form-control col custom-select" data-bind="value: sortBy" aria-label="Sort Order">
+                    <option value="dateCreated">Date uploaded</option>
```

Comment:
Order of sorting is not very clear. Change to something like `Recently uploaded` and `Recently modified`